validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/synccommittee/SyncCommitteeScheduledDutiesTest.java

```diff
@@ -78,6 +79,15 @@ void shouldSubscribeToSubnets() {
                     70, Set.of(2, 6), PERIOD_END_EPOCH.increment())));
   }
 
+  @Test
+  void shouldNotSubscribeToSubnetsWithoutValidatorsPresent() {
+    final SyncCommitteeScheduledDuties duties = validBuilder().build();
+
+    duties.subscribeToSubnets();
+
+    verifyNoInteractions(validatorApiChannel);
```

Comment:
We probably should confirm we don't call `chainHeadTracker.getCurrentChainHead` here either since we have no validators.